packages/core/src/_common/utility.js

```diff
@@ -52,11 +52,46 @@ const copyToClipboard = text => {
     document.execCommand('copy');
     textField.remove();
 };
+// eu countries to support
+const eu_countries = [
+    'it',
+    'de',
+    'fr',
+    'lu',
+    'gr',
+    'mf',
+    'es',
+    'sk',
+    'lt',
+    'nl',
+    'at',
+    'bg',
+    'si',
+    'cy',
+    'be',
+    'ro',
+    'hr',
+    'pt',
+    'pl',
+    'lv',
+    'ee',
+    'cz',
+    'fi',
+    'hu',
+    'dk',
+    'se',
+    'ie',
+    'im',
+    'gb',
+];
+// check if client is from EU
+const isEuCountry = clients_country => eu_countries.includes(clients_country);
```

Comment:
It doesn't have to be the client's country, we can update for accuracy.

```suggestion
const isEuCountry = country => eu_countries.includes(country);
```